diff --git a/src/main/antlr4/Rust.g4 b/src/main/antlr4/Rust.g4
index 9a1ab55..f917393 100644
--- a/src/main/antlr4/Rust.g4
+++ b/src/main/antlr4/Rust.g4
@@ -798,10 +798,10 @@ INNER_DOC_COMMENT : '//!' ~[\n]*
 
 // HELPER DEFINITIONS:
 
-WS : [ \t\r\n]+ -> channel(HIDDEN) ; // skip spaces, tabs, newlines
-OTHER_LINE_COMMENT : '//' ~[\n] * -> channel(HIDDEN) ;
-OTHER_BLOCK_COMMENT : '/*' (~[*] | ('*'+ ~[*/]))* '*'+ '/' -> channel(HIDDEN) ;
-SHEBANG_LINE : {at_beginning_of_file()}? '#!' ~[\n]* '\n' -> channel(HIDDEN) ;
+WS : [ \t\r\n]+ -> skip ; // skip spaces, tabs, newlines
+OTHER_LINE_COMMENT : '//' ~[\n] * -> skip ;
+OTHER_BLOCK_COMMENT : '/*' (~[*] | ('*'+ ~[*/]))* '*'+ '/' -> skip ;
+SHEBANG_LINE : {at_beginning_of_file()}? '#!' ~[\n]* '\n' -> skip ;
 
 BINDIGIT : [0-1_] ;
 DECDIGIT : [0-9] ;
